ci-operator/step-registry/openshift-qe/kube-burner/openshift-qe-kube-burner-chain.yaml

```diff
@@ -0,0 +1,8 @@
+
+chain:
+  as: openshift-qe-kube-burner
+  steps:
+  - ref: openshift-qe-kube-burner
+  - ref: openshift-qe-write-perfscale-results
+  documentation: |-
+    This workflow executes cluster density v2 and all node density workloads using kube-burner ocp wrapper
```

Comment:
lets update the documentation here to be general, could also use a link to [kube burner ocp](https://cloud-bulldozer.github.io/kube-burner/latest/ocp/) or e2e-benchmarking